platform/features/timeline/src/actions/CompositionColumn.js

```diff
@@ -41,7 +42,9 @@ define([], function () {
     CompositionColumn.prototype.value = function (domainObject) {
         var model = domainObject.getModel(),
             composition = model.composition || [];
-        return (composition[this.index]) || "";
+
+        return composition.length > this.index ?
+            this.idMap[composition[this.index]] : "";
```

Comment:
There are a few odd edge cases where this would behave differently (if a domain object had id "undefined"; or, slightly more plausibly, if the value in `idMap` was 0) so I'd prefer to keep the current form